ivy/functional/frontends/paddle/nn/functional/common.py

```diff
@@ -4,6 +4,30 @@
 from ivy.functional.frontends.paddle.func_wrapper import to_ivy_arrays_and_back
 
 
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def alpha_dropout(x, p=0.5, training=True, name=None):
+    if training:
+        alpha = 1.67326
+        scale = 1.0507
+        keep_prob = 1 - p
+        random_values = ivy.random_normal(shape=x.shape, dtype=x.dtype)
```

Comment:
With randomization we won't be able to get exact results here you can refer to how other `Selu` activation functions are implemented in frontends.